ibc/types.go

```diff
@@ -44,6 +44,8 @@ type ChainConfig struct {
 	GasPrices string `yaml:"gas-prices"`
 	// Adjustment multiplier for gas fees.
 	GasAdjustment float64 `yaml:"gas-adjustment"`
+	// Default gas limit for transactions. May be empty, "auto", or a number.
+	Gas string `yaml:"gas"`
```

Comment:
If gas is empty, should we default to `auto` rather than having to specify it?

```go
Gas string `yaml:"gas" default:"auto"`
```
or something like that

Reply:
so it turns out that the yaml parser doesn't actually interpret the `default` flag 🙃 
I added it anyway fwiw, and I added a test with it; I guess we could go set `gas: auto` to a bunch of the bigger chains, like i did with gaia here